test/Spec.hs

```diff
@@ -1 +1,26 @@
-{-# OPTIONS_GHC -F -pgmF tasty-discover #-}
+module Main where
+
+import Hedgehog (Group (..), checkParallel)
+import Test.Hspec (hspec)
+
+import Test.DecisionSpec (decisionMonoidMempty, decisionSemigroupAssoc)
+import Test.Golden (goldenSpec)
+import Test.QuestionSpec (yesNoPromptSpec)
+import Test.TomlSpec (tomlProp)
+
+
+main :: IO ()
+main = do
+    hspec yesNoPromptSpec
+    hspec goldenSpec
```

Comment:
You can write just:
```haskell
hspec $ do
    yesNoPromptSpec
    goldenSpec
```

Though it's one more extra line... Alternative you create list of specs  with something like:

```haskell
hspec $ sequence_ [yesNoPromptSpec, goldenSpec]
```